doc/Extensions/SNMP-Trap-Handler.md

```diff
@@ -159,16 +159,19 @@ So what value should you type in the commands below? Oddly enough, simply supply
 ### Event logging
 
 You can configure generic event logging for snmp traps.  This will log
-an event of the type trap for received traps. These events can be utilized for alerting.
+an event of the type trap for received traps. These events can be used for alerting.
+By default, only the TrapOID is logged. Buf you can enable the "detailed" variant,
+and all the data received with the trap will be logged.
 
 In config.php
 
 ```php
-$config['snmptraps']['eventlog'] = 'unhandled';
+$config['snmptraps']['eventlog'] = 'unhandled'; //default value
+$config['snmptraps']['eventlog_detailed'] = 'false'; //default value
```

Comment:
this should be in the dynamic config instead

Reply:
Never played with it. Will have a look.

Reply:
Dynamic config done. Still not very comfortable with it but it looks ok.